docs/examples/hsv_example.ipynb

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
I would expect the example to have some more description.
Now its like: https://www.youtube.com/watch?v=MAlSjtxy5ak
At least it would be nice to describe what we achieved with those transformations, and how different parameters influence the result.

Reply:
Done